scripts/invoke_increment.sh

```
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+sncast invoke \
+    --contract-address 0x033c9817a4f4f7941451295fbcd5f34c589115af8716991f21a6afab5b3e3b57 \
+    --function "increment" \
+    --fee-token eth \
```

Shellcheck (0.10.0)

[error] Couldn't parse this escaped char. Fix to allow more checks.

(SC1073)